tests/unit/jobserver/models/test_paired_fields.py

```diff
@@ -0,0 +1,301 @@
+import pytest
+from django.db import IntegrityError
+from django.utils import timezone
+
+from ....factories import UserFactory
```

Comment:
A comment I got from the dearly departed George was that `tests.factories` is probably more readable than all those dots

Reply:
Ah, okay. Some of the modules in `tests/unit/jobserver/models` use `....factories`; some use `tests.factories`. I agree that the latter is more readable, but I thought that sticking to convention was more important. Turns out that there isn't a convention 🙂 Thank you for mentioning it.